### PR TITLE
Add analytics

### PR DESCRIPTION
**Pull request by alexwykoff**
Initial implementation of analytics. 
Added tracking event for Wordpress verification.

### CODE REVIEW

**Comment by alexwykoff** [comment]
Oh my... looks like I have some fixing to do.

**Comment by ayumi** [comment]
can you rebase this on `master`?

File: app/assets/javascripts/analytics.js [hunk 1]
```diff
@@ -14,17 +14,16 @@
...
     g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
   })();
-</script>
+
 <!-- End Piwik Code -->
 
 
-document.addEventListener('click', function(e) {
+//trying to track if a user clicked on the verify button for a Wordpress flow
+$(document).addEventListener('click', function(e) {
```
Comment (ayumi) [inline app/assets/javascripts/analytics.js]: no need to wrap this in jQuery `$` i think?

File: app/assets/javascripts/analytics.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+//Piwik analytics shim
+//data is sent to https://analytics.brave.com
+
+<!-- Piwik -->
+
+  var _paq = _paq || [];
```
Comment (ayumi) [inline app/assets/javascripts/analytics.js]: minor: indentation left 2 sp

File: app/views/layouts/application.html.slim [hunk 3]
```diff
@@ -18,7 +18,8 @@ html
       - if content_for(:content_form_errors)
         .alert.alert-warning= I18n.t("activerecord.shared.errors")
         = yield(:content_form_errors)
-            
+
     = content_for?(:content) ? yield(:content) : yield
 
     = render("footer")
+    = javascript_include_tag('analytics')
```
Comment (ayumi) [inline app/views/layouts/application.html.slim]: this is not necessary as app/assets/javascripts does a `require_tree .` so all JS there is available everywhere

File: app/assets/javascripts/analytics.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    _paq.push(['setTrackerUrl', u+'piwik.php']);
+    _paq.push(['setSiteId', '6']);
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
+  })();
+
+<!-- End Piwik Code -->
+
+
+//trying to track if a user clicked on the verify button for a Wordpress flow
```
Comment (ayumi) [inline app/assets/javascripts/analytics.js]: better if you move this comment below directly next to the `if (target.dataset.trackWordpress === 1){`

File: app/assets/javascripts/analytics.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+$(document).addEventListener('click', function(e) {
```
Comment (ayumi) [inline app/assets/javascripts/analytics.js]: for publishers i think we should prefer vanilla JS
```
document.body.addEventListener('click', function (e) {
```

File: app/views/layouts/application.html.slim (hunk 3 above)
Comment (alexwykoff) [reply app/views/layouts/application.html.slim]: ah, that's my misunderstanding. i had thought we wanted non-render js to load afterwards.
if all of app/assets/javascripts is being pulled in anyways then i can remove it 😄

**Comment by alexwykoff** [comment]
New PR generated based on master and w/ edits, I'll close this one out.

**Pull request closed without merging**